### ISSUE TITLE
Mouse wheel scrolling has gotten slower

### ISSUE THREAD

**Issue by bpasero** [microsoft/vscode#204086]
I see a noticable slowdown in scrolling speed when comparing latest builds with previous builds. I am quite sensitive to this and could not find a relevant setting. I am reporting this because I know we did make changes in this area.

I use an external Logitech mouse and here is the result of pushing the wheel down by 2 ticks (my wheel operates in a mode where I can scroll in discrete steps):

![image](https://github.com/microsoft/vscode/assets/900690/17e91f75-ea53-4392-93dc-ae505ecf6d57)

Compared to before:

![image](https://github.com/microsoft/vscode/assets/900690/3163bb8b-847d-4852-b243-9e97b637ba11)

My settings:

```json
// Editor
"editor.fontFamily": "Fira Code",
"editor.fontSize": 13,
"editor.fontLigatures": true,
"editor.wordWrap": "off",
"editor.dragAndDrop": false,
"editor.folding": true,
"editor.formatOnSave": true,
"editor.codeLens": true,
"editor.renderWhitespace": "none",
"editor.snippetSuggestions": "none",
"editor.minimap.enabled": false,
"editor.find.globalFindClipboard": false,
"editor.inlineSuggest.enabled": true,
"editor.smoothScrolling": false,
"editor.suggestSelection": "first",
"editor.suggest.localityBonus": true,
"editor.hover.delay": 800,
"editor.guides.indentation": false,
"editor.renderControlCharacters": true,
"editor.gotoLocation.multipleDefinitions": "goto",
"diffEditor.renderIndicators": false,
"diffEditor.experimental.showMoves": true,
"diffEditor.useInlineViewWhenSpaceIsLimited": false,
```

**Comment by bpasero** [comment]
Oh, this is very likely https://github.com/microsoft/vscode/commit/6ab0c535f92306ace1aca3998ac30f9d06a43b57

I am using `window.zoomLevel: 1`.

**Comment by alexdima** [comment]
6ab0c53 is about fixing smooth scrolling post https://github.com/microsoft/vscode/pull/200776. The root cause of the slowdown you're experiencing is https://github.com/microsoft/vscode/pull/200776 which is a bug fix to https://github.com/microsoft/vscode/issues/146403 .

TL;DR with 1.66 scrolling became twice as fast when on high dpi monitors. With 1.86, the intent is to return scrolling to the speeds of 1.65.

**Comment by bpasero** [comment]
`"editor.mouseWheelScrollSensitivity": 1.5` saves me!
